packages/ui/src/forum/components/threads/ThreadList.tsx

```diff
@@ -79,11 +75,11 @@ export const ThreadList = ({ threads, onSort, isLoading, isArchive, page, pageCo
         <Loading />
       ) : (
         <List as="div" isArchive={isArchive}>
-          {pagination}
+          {page && pageCount && setPage && <Pagination pageCount={pageCount} handlePageChange={setPage} page={page} />}
```

Comment:
I thought we could prevent this by https://github.com/Joystream/pioneer/issues/1439. But it seems that TS might complain on undefined props. AFAICS, the interface is:

```
interface PaginationProps {
  pageCount?: number
  handlePageChange: (page: number) => void
  page?: number
}
```
so this should work:
```ts
{setPage && <Pagination pageCount={pageCount} handlePageChange={setPage} page={page} />
```
Or we could just make `handlePageChange` optional as it is optional in the component below.